scenarios/community/vmware/ansible.yml

```diff
@@ -0,0 +1,40 @@
+netcommon:
```

Comment:
does vmware depend on netcommon? if not, we can delete this

Reply:
ok